spring-test/testcontainer/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.camunda</groupId>
+    <artifactId>spring-boot-starter-camunda-test-pom</artifactId>
+    <version>8.6.7-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>spring-boot-starter-camunda-test-testcontainer</artifactId>
```

Comment:
A bit unfortunate naming inconsistency with `qa/testcontainers`, missing the plural `s`. However for better pre 8.6 migration experience I would keep this artifacts name singular. Wdyt?